hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/client/TokenClient.java

```diff
@@ -650,6 +651,40 @@ public NetworkTransactionResponse updateTokenFeeSchedule(
         return response;
     }
 
+    public NetworkTransactionResponse rejectFungibleToken(List<TokenId> tokenIds, AccountId ownerId, PrivateKey privateKey, ExpandedAccountId payer) {
+        var tokenRejectTransaction =
+                getFungibleTokenRejectTransaction(tokenIds).setOwnerId(ownerId).setTransactionMemo(getMemo("Reject Fungible token"));
+        var response = executeTransactionAndRetrieveReceipt(
+                tokenRejectTransaction, privateKey == null ? null : KeyList.of(privateKey), payer);
```

Comment:
I don't think having Carol pay is necessary for our tests. We can make the operator pay which has more funds and simplifies the setup. Same for other.
```suggestion
        var response = executeTransactionAndRetrieveReceipt(
                tokenRejectTransaction, privateKey == null ? null : KeyList.of(privateKey));
```

Reply:
fixed